run.py

```diff
@@ -16,6 +16,6 @@
         logger.info('* Cloud Foundry port "{}"'.format(port))
         logger.info('* Cloud Foundry protocol "{}"'.format(protocol))
     else:
-        port = os.getenv('FS_DEV_PORT', 5001)
+        port = os.getenv('FS_DEV_PORT', 8080)
```

Comment:
I think we should change this as frontstage should be the one service that does use 80 or 8080. I sent out a proposal probably 1 - 2 months ago and included RM guys too. The docker-compose files have now been updated to this. So unless it clashes with other services I think we should update to a standard port for web UI.